book/forms.md

```diff
@@ -70,6 +70,18 @@ a great starting point for transforming our toy browser into an
 application platform---our goal for these next few chapters. Let's get
 started implementing all that!
 
+::: {.further}
+HTML forms first standardized in [HTML+][htmlplus], which also
+proposed tables, mathematical equations, and text that wraps around
+images. Amazingly, all three of these technologies survive, but in
+totally different standards: tables in [RFC 1942][rfc1942], equations
+in [MathML][mathml], and floating images in [CSS 1.0][css1].
+:::
+
+[htmlplus]: https://www.w3.org/MarkUp/htmlplus_paper/htmlplus.html
+[rfc1942]: https://datatracker.ietf.org/doc/html/rfc1942
```

Comment:
And https://drafts.csswg.org/css-tables-3/

Reply:
I'm not sure HTML plus actually ever standardized, say, what tables were supposed to look like, so I don't know if CSS 3 is the right link here. (Plus I kind of like the asymmetry of tables, math, and figures all ending up in different WGs.)

Reply:
ok